src/microbe_stage/ProcessSpeedInformation.cs

```diff
@@ -37,7 +37,11 @@ public ProcessSpeedInformation(BioProcess process)
     public IReadOnlyDictionary<Compound, float> FullSpeedRequiredEnvironmentalInputs =>
         WritableFullSpeedRequiredEnvironmentalInputs;
 
-    public IEnumerable<KeyValuePair<Compound, float>> Outputs => WritableOutputs;
+    public IEnumerable<KeyValuePair<Compound, float>> Outputs =>
+        WritableOutputs.Where(p => !p.Key.IsEnvironmental);
+
+    public IEnumerable<KeyValuePair<Compound, float>> EnvironmentalOutputs =>
+        WritableOutputs.Where(p => p.Key.IsEnvironmental);
```

Comment:
Why were these needed?

Reply:
These are not useful as of now because there is no environmental output, but it will start to be useful when some processes like photosynthesis will start rejecting environmental compounds. Yet, the previous infrastructure assumes there is no environmental outputs, so I filtered them out to keep things working.

Having access to Environmental ones rather than the full list is just sugar to mimick inputs and help readability.

Reply:
I'd like a comment here detailing this aspect so that when someone is fixing this up properly they'll have the necessary context to know to change these back.